### ISSUE TITLE
az monitor diagnostic-settings create help

### ISSUE THREAD

**Issue by sudeep1607** [Azure/azure-cli#16742]
Is there a way to specify the resource groups for both the resource for which diagnostic settings is being created and the workspace which this diagnostic setting is being linked to while using the below command

az monitor diagnostic-settings create

Currently i can see only one option --resource-group. But if the resource and log analytics workspace are in different resource groups how to specify the resource groups for both separately?

**Comment by yonzhan** [comment]
monitor

**Comment by kairu-ms** [comment]
You can use the ID of workspace for `--workspace` argument

```example
--workspace /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/microsoft.OperationalInsights/workspaces/wprkspace000003
```

**Comment by nextgen1607** [comment]
@kairu-ms Yes, we can do that. But it would be nice to just provide the workspace resource group instead of giving the entire path starting from /subscriptions
